src/util/style.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var util = module.exports = {};
+
+util.computePaddingValues = function( width, height, paddingObject, relativeTo ){
+  // Assuming percentage is number from 0 to 1
+  if( paddingObject.units === '%' ) {
+    switch( relativeTo ) {
+      case 'width':
+        return width > 0 ? paddingObject.pfValue * width : 0;
+      case 'height':
+        return height > 0 ? paddingObject.pfValue * height : 0;
+      case 'average':
+        return ( width > 0 ) && ( height > 0 ) ? paddingObject.pfValue * ( width + height ) / 2 : 0;
+      case 'min':
+        return ( width > 0 ) && ( height > 0 ) ? ( ( width > height ) ? paddingObject.pfValue * height : paddingObject.pfValue * width ) : 0;
+      case 'max':
+        return ( width > 0 ) && ( height > 0 ) ? ( ( width > height ) ? paddingObject.pfValue * width : paddingObject.pfValue * height ) : 0;
+      default:
+        return 0;
+    }
+  } else if( paddingObject.units === 'px' ) {
+    return paddingObject.pfValue;
+  } else {
+    return 0;
+  }
+};
+
+util.getIndexedStyle = function( node, property, subproperty, index ){
```

Comment:
It might make sense to put this in the style / stylesheet object instead of util, as it's style related rather than general object util